introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello My Name is Trinh a.k.a Mastaboywonder
 My favorite Code School path is HTML
+Deadly Skills
+*Git
+*HTML
+*Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?